src/bundle/Resources/public/js/scripts/admin.context.menu.js

```diff
@@ -0,0 +1,53 @@
+(function(global, doc, eZ) {
+    const adapatItemsContainer = doc.querySelector('.ibexa-context-menu');
+
+    if (adapatItemsContainer) {
+        const menuButtons = [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item > .ibexa-btn:not(.ibexa-btn--more)')];
+        const popupMenuElement = adapatItemsContainer.querySelector('.ibexa-popup-menu');
+        const showPopupButton = adapatItemsContainer.querySelector('.ibexa-btn--more');
+        const adaptiveItems = new eZ.core.AdaptiveItems({
+            items: [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item:not(.ibexa-context-menu__item--more)')],
+            selectorItem: adapatItemsContainer.querySelector('.ibexa-context-menu__item--more'),
+            itemHiddenClass: 'ibexa-context-menu__item--hidden',
+            container: adapatItemsContainer,
+            getActiveItem: () => {
+                return adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item')[0];
+            },
+            onAdapted: (visibleItems, hiddenItems) => {
+                const hiddenButtonsIds = [...hiddenItems].map((item) => item.querySelector('.ibexa-btn').id);
+
+                popupMenu.toggleItems((popupMenuItem) => !hiddenButtonsIds.includes(popupMenuItem.dataset.relatedButtonId));
+            },
+        });
+        const popupMenu = new eZ.core.PopupMenu({
+            popupMenuElement,
+            triggerElement: showPopupButton,
+            onItemClick: (event) => {
+                const { relatedButtonId } = event.currentTarget.dataset;
+                const button = doc.getElementById(relatedButtonId);
+
+                button.click();
+            },
+            position: () => {
+                popupMenuElement.style.right = 0;
+            },
```

Comment:
And proper styles in CSS. Would it work? 
```suggestion
            position: () => {},
```